e2e/test/scenarios/embedding-sdk/interactive-question.cy.spec.ts

```diff
@@ -44,32 +39,20 @@ describeSDK("scenarios > embedding-sdk > interactive-question", () => {
     );
 
     cy.signOut();
+  });
 
-    cy.intercept("GET", "/api/card/*").as("getCard");
-    cy.intercept("GET", "/api/user/current").as("getUser");
-    cy.intercept("POST", "/api/card/*/query").as("cardQuery");
-
+  it("should show question content", () => {
     visitInteractiveQuestionStory();
 
-    cy.wait("@getUser").then(({ response }) => {
-      expect(response?.statusCode).to.equal(200);
-    });
-
-    cy.wait("@getCard").then(({ response }) => {
-      expect(response?.statusCode).to.equal(200);
+    getSdkRoot().within(() => {
+      cy.findByText("Product ID").should("be.visible");
+      cy.findByText("Max of Quantity").should("be.visible");
     });
   });
 
-  it("should show question content", () => {
-    cy.get("#metabase-sdk-root")
-      .should("be.visible")
-      .within(() => {
-        cy.findByText("Product ID").should("be.visible");
-        cy.findByText("Max of Quantity").should("be.visible");
-      });
-  });
-
   it("should not fail on aggregated question drill", () => {
+    visitInteractiveQuestionStory();
```

Comment:
We were visiting the question twice on the pre-defined collection test, once in `beforeEach` and once in the test. I'm fixing this in this PR, as our test also needs to visit a different story id. Calling `visitInteractiveQuestionStory()` manually in each tests helps to specify the arguments for the story.

<img src="https://github.com/user-attachments/assets/c62210ce-7153-44e0-8136-07a142b5cf46" width="300">